app/src/main/java/io/intrepid/contest/screens/adminstatus/AdminStatusPresenter.java

```diff
@@ -0,0 +1,75 @@
+package io.intrepid.contest.screens.adminstatus;
+
+import android.support.annotation.NonNull;
+
+import java.util.List;
+
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BasePresenter;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Entry;
+import io.reactivex.disposables.Disposable;
+import timber.log.Timber;
+
+class AdminStatusPresenter extends BasePresenter<AdminStatusContract.View> implements AdminStatusContract.Presenter {
+
+    private List<Entry> entries;
```

Comment:
It looks like we're only using this in one place, we can just make this a local value.